src/Sunny.jl

```diff
@@ -1,6 +1,7 @@
 module Sunny
 
 using LinearAlgebra
+using LinearMaps
```

Comment:
Please use `import LinearMaps` instead, and then qualify references, e.g. `LinearMaps.x`.